core/src/main/java/io/grpc/internal/DelayedClientTransport.java

```diff
@@ -252,13 +277,26 @@ public final boolean hasPendingStreams() {
     }
   }
 
+  public final boolean hasUncommittedStreams() {
+    synchronized (lock) {
+      return !toCheckCompletionStreams.isEmpty();
+    }
+  }
+
   @VisibleForTesting
   final int getPendingStreamsCount() {
     synchronized (lock) {
       return pendingStreams.size();
     }
   }
 
+  @VisibleForTesting
+  final int getUncommittedStreamsCount() {
+    synchronized (lock) {
+      return toCheckCompletionStreams.size();
```

Comment:
What about using the same name for `toCheckCompletionStreams` and `uncommittedStreams`?